pCrunch/batch_processing.ipynb

```diff
@@ -26,8 +26,8 @@
     "# %matplotlib widget\n",
     "# ROSCO toolbox modules \n",
     "from ROSCO_toolbox import utilities as rosco_utilities\n",
-    "# WEIS modules\n",
-    "from weis.aeroelasticse.Util import FileTools\n",
+    "# WISDEM modules\n",
+    "from wisdem.aeroelasticse.Util import FileTools\n",
```

Comment:
I think this was an early test script and not used
@JakeNunemaker : Does this need to be updated or removed from the pCrunch branch?

Reply:
This is also a leftover. I can delete if we need to otherwise I was leaving these things in there to be updated outside of WEIS when there was time/budget

Reply:
This jupyter notebook was made to provide some working examples of the functionalities of pCrunch, how the API works, etc... If it isn't too heavy of a lift, I don't think it would hurt to get this (or something similar) updated for the overhaul.

Reply:
I've added an updated example notebook that covers the basic usage and new API of the new pCrunch but I think it'll take a while to fully recreate this notebook (there is a lot there!).

Reply:
Seems like there is more to do in pCrunch, but it doesn't have to all get done for this PR.   I think the example you provided should be good for now.